packages/eas-cli/src/commands/build/index.ts

```diff
@@ -208,3 +217,71 @@ export default class Build extends EasCommand {
     };
   }
 }
+
+async function handleDeprecatedEasJsonAsync(
+  projectDir: string,
+  nonInteractive: boolean
+): Promise<void> {
+  const reader = new EasJsonReader(projectDir);
+  const profileNames = await reader.getBuildProfileNamesAsync();
+  const platformAndProfileNames: [Platform, string][] = profileNames.flatMap(profileName => [
+    [Platform.ANDROID, profileName],
+    [Platform.IOS, profileName],
+  ]);
+
+  const deprecatedProfiles: [Platform, string][] = [];
+
+  for (const [platform, profileName] of platformAndProfileNames) {
+    const buildProfile = await reader.getBuildProfileAsync(platform, profileName);
```

Comment:
consider reading just raw json, with this change if someone has an invalid profile in eas.json it won't be possible to build for any other profile too

Reply:
Acknowledged, won't fix :D